### PR TITLE
Konstantinos / [FE Deriv.com squad] [EU] Review and update the design of Dtrader page

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [x] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/_dhero.js [hunk 1]
```diff
@@ -231,14 +235,14 @@ const DHero = ({
...
     image_name,
-    is_mobile,
     join_us_for_free,
     go_to_live_demo,
     Logo,
 }) => {
     const data = useStaticQuery(query)
     const getLinkType = () => (image_name === 'dbot' ? 'dbot' : 'deriv_app')
     const handleSignup = useHandleSignup()
+    const { is_eu_country } = React.useContext(DerivStore)
```
Comment (prince-deriv) [inline src/components/custom/_dhero.js:245]: Please use the `useCountryRule` instead

File: src/pages/dtrader/index.tsx [hunk 2]
```diff
@@ -68,11 +80,42 @@ const trading = [
...
+        image_name: 'how_you_want',
+        image_alt: localize('Customise your trade types'),
+    },
+]
 const PlatformContainer = styled.div`
     padding: 8rem 0;
 `
 const Dtrader = () => {
     const [is_mobile, setMobile] = useState(false)
+    const { is_eu_country } = React.useContext(DerivStore)
```
Comment (prince-deriv) [inline src/pages/dtrader/index.tsx:118]: Here as well

File: src/components/custom/_dhero.js (hunk 1 above)
Comment (konstantinosG-deriv) [reply src/components/custom/_dhero.js:245]: Ok

File: src/pages/dtrader/_dtrader-tabs.tsx [hunk 3]
```diff
@@ -270,8 +271,15 @@ class DtraderTabs extends React.Component {
...
                         playsInline
                     >
-                        <source src="/Dtrader_GIF.mp4" type="video/mp4" />
-                        <source src="/Dtrader_GIF.webm" type="video/webm" />
+                        <ROW>
+                            <>
+                                <source src="/Dtrader_GIF.mp4" type="video/mp4" />
+                                <source src="/Dtrader_GIF.webm" type="video/webm" />
+                            </>
+                        </ROW>
```
Comment (prince-deriv) [inline src/pages/dtrader/_dtrader-tabs.tsx:279]:
```suggestion
                        <ROW>
                                <source src="/Dtrader_GIF.mp4" type="video/mp4" />
                                <source src="/Dtrader_GIF.webm" type="video/webm" />
                        </ROW>
```

File: src/pages/dtrader/index.tsx [hunk 4]
```diff
@@ -68,11 +80,42 @@ const trading = [
         image_alt: localize('Customise your trade types'),
     },
 ]
+
+const tradingEU = [
```
Comment (prince-deriv) [inline src/pages/dtrader/index.tsx:84]:
```suggestion
const trading_eu = [
```

File: src/pages/dtrader/_dtrader-tabs.tsx [hunk 5]
```diff
@@ -270,8 +271,15 @@ class DtraderTabs extends React.Component {
                         muted
                         playsInline
                     >
-                        <source src="/Dtrader_GIF.mp4" type="video/mp4" />
-                        <source src="/Dtrader_GIF.webm" type="video/webm" />
+                        <ROW>
+                            <>
```
Comment (ShuvoHabib) [inline src/pages/dtrader/_dtrader-tabs.tsx:275]: This empty tag not needed

File: src/pages/dtrader/_dtrader-tabs.tsx (hunk 3 above)
Comment (konstantinosG-deriv) [reply src/pages/dtrader/_dtrader-tabs.tsx:279]: If I remove the fragment I get this error: This JSX tag's 'children' prop expects a single child of type 'ReactElement<any, string | JSXElementConstructor<any>>', but multiple children were provided.

File: src/pages/dtrader/_dtrader-tabs.tsx (hunk 5 above)
Comment (konstantinosG-deriv) [reply src/pages/dtrader/_dtrader-tabs.tsx:275]: If I remove the fragment I get this error: This JSX tag's 'children' prop expects a single child of type 'ReactElement<any, string | JSXElementConstructor>', but multiple children were provided.

**Pull request closed without merging**